### PR TITLE
OrgEditor

### PR DESCRIPTION
**Pull request by MisterY**
## Connection with issue(s)

Resolve issue #474 

## Testing and Review Notes

Introducing the OrgEditor in order to handle .org note type and serialization.

## To Do

Will polish the rough edges and introduce additional functionality separately.

### CODE REVIEW

File: lib/editors/org_editor.dart [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/*
+Copyright 2020-2021 Vishesh Handa <me@vhanda.in>
```
Comment (vHanda) [inline lib/editors/org_editor.dart:2]: You probably want to change the name :D

Comment (MisterY) [reply lib/editors/org_editor.dart]: I did it for this file. Not even sure how this works. :@

**Comment by vHanda** [comment]
Could you please fix the problems and merge it. I think the pubspec changes are no longer required as I've ported it to Flutter 2.

**Comment by MisterY** [comment]
> Could you please fix the problems and merge it. I think the pubspec changes are no longer required as I've ported it to Flutter 2.

Oh, great! Let me then also upgrade my SDK and Flutter, and review this again.

**Pull request merged**